src/repositories_rust_libraries.tf

```diff
@@ -0,0 +1,101 @@
+#####################################################################
+#
+# Repository settings for Arrow Services
+#
+#####################################################################
+locals {
+  lib = {
+    repository_prefix  = "lib"
+    description_prefix = "Arrow Libraries"
+
+    # List of webhooks to apply for all services repositories
+    webhooks = {
+      "discord" = {
+        events = [
+          "pull_request",
+          "pull_request_review",
+          "pull_request_review_comment",
+          "pull_request_review_thread"
+        ]
+        configuration = {
+          url = local.discord_services_integration_url
+        }
+      }
+    }
+
+    lib_files = {
```

Comment:
Can drop the lib_ prefix since it's defined inside the lib map already.

Reply:
Fixed in https://github.com/Arrow-air/tf-github/pull/13/commits/92d3024b0ec02843802f9c6da2a1eeb6a9e3a283